doorstop/settings.py

```diff
@@ -30,6 +30,7 @@
 CHECK_LEVELS = True  # validate document levels during validation
 CHECK_REF = True  # validate external file references
 CHECK_CHILD_LINKS = True  # validate reverse links
+CHECK_CHILD_LINKS_STRICT = False  # Strictly validate reverse links from all the child documents
```

Comment:
To me, a comment like "require reverse links from every child document" is more clear.